src/styles/mixins/utils.scss

```diff
@@ -0,0 +1,48 @@
+@use "../variables" as var;
```

Comment:
Totally open to talking about this. Tell you what: I've got a bunch of subsequent commits in this work chain that make modifications to this module, so it would be easier to rename later (I've spent enough overhead time in git wrangling lately...). Happy to stand back and reassess once the chain is landed...